google_play_description.txt

```diff
@@ -1,5 +1,8 @@
 Explore beyond the reaches of the Internet with Trail Sense.
```

Comment:
Can you undo the changes to this file? I would like this to match exactly with the Google Play Store description, and I prefer the order / text of the current description.

Reply:
(and the F-Droid description should be updated to match the current Google Play Store description)